frameworks/kafka/tests/test_availability.py

```diff
@@ -29,10 +29,10 @@ def setup_module(module):
 
     sdk_install.uninstall(SERVICE_NAME, PACKAGE_NAME)
 
-    sdk_install.install(
-        PACKAGE_NAME,
-        DEFAULT_BROKER_COUNT,
-        service_name=SERVICE_NAME,
+    config.install(
+        package_name=config.PACKAGE_NAME,
```

Comment:
Perhaps we should retain consistency in the `config.install` calls by either specifying argument names or just dropping them (as we've done in most calls) ?

Reply:
There is a difference between argument order in 0.30.X and `master` so I added argument names where necessary. I'll clean it up when I backport the similar changes for Elastic.